homeassistant/components/recorder/table_managers/__init__.py

```diff
@@ -1 +1,15 @@
 """Managers for each table."""
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ..core import Recorder
+
+
+class BaseTableManager:
+    """Base class for table managers."""
+
+    def __init__(self, recorder: "Recorder") -> None:
```

Comment:
There is a bit of duplication in these which would be nice to reduce in a future PR